packages/react-dom/src/__tests__/ReactTestUtilsActUnmockedScheduler-test.js

```diff
@@ -23,48 +24,43 @@ function clearLog() {
   }
 }
 
-function render(el, dom) {
-  ReactDOM.render(el, dom);
-}
-
-function unmount(dom) {
-  ReactDOM.unmountComponentAtNode(dom);
-}
-
 beforeEach(() => {
+  prevActGlobal = global.IS_REACT_ACT_ENVIRONMENT;
+  global.IS_REACT_ACT_ENVIRONMENT = true;
```

Comment:
Wait why is this needed now and not before?

Reply:
With concurrent rendering there is a new path hit during the setState https://github.com/facebook/react/blob/main/packages/react-reconciler/src/ReactFiberWorkLoop.js#L4006

isConcurrentActEnvironment logs and error if the global is not set

Reply:
Oh, right, this is required to use external act.